### PR TITLE
Add rescaled labels and packaging arrows for light sources

### PR DESCRIPTION
**Pull request by matteason**
The packaging arrows are a bit of a hybrid of energy label and internet arrow but I've decided to treat them more like an energy label. PDF will be a better format for designers, we can't ask the price height question (as they're for the packaging and there's no prescribed height, just 'legible'), there's additional guidance needed around the colour/black and white question and the little SVG icons for the orientation question are different

### CODE REVIEW

**Comment by matteason** [comment]
Just noticed the failing tests, I'll fix those up

File: src/main/java/uk/gov/beis/els/service/DocumentRendererService.java [hunk 1]
```diff
@@ -66,13 +65,7 @@ public ResponseEntity processImageResponse(ProcessedInternetLabelDocument proces
...
 
   private String generatePdfFilename(ProcessedEnergyLabelDocument processedDocument) {
-    String filename;
-    if (StringUtils.isBlank(processedDocument.getSupplierName()) || StringUtils.isBlank(processedDocument.getModelName())) {
-       filename = processedDocument.getProductMetadata().getProductFileName() + ".pdf";
-    } else {
-      filename = String.format(processedDocument.getProductMetadata().getProductFileName() + " - %s - %s.pdf",
-          processedDocument.getSupplierName(), processedDocument.getModelName());
-    }
+    String filename = processedDocument.getDocument().title()+".pdf";
```
Comment (matteason) [inline src/main/java/uk/gov/beis/els/service/DocumentRendererService.java:68]: Went for this in the end as my attempts to move it into a util function weren't very elegant

File: src/main/java/uk/gov/beis/els/categories/lamps/controller/LampsController.java [hunk 2]
```diff
@@ -64,22 +69,23 @@ public ModelAndView renderLamps(@ModelAttribute("form") LampsForm form) {
...
-    else {
-      return documentRendererService.processPdfResponse(lampsService.generateHtml(form, LampsService.LEGISLATION_CATEGORY_CURRENT));
-    }
+    return doIfValid(form, bindingResult, (category -> documentRendererService.processPdfResponse(lampsService.generateHtml(form, category))));
   }
 
   @PostMapping(value = "/lamps", params = "mode=INTERNET")
   @ResponseBody
   public Object handleInternetLabelLampsSubmit(@Validated(InternetLabellingGroup.class) @ModelAttribute("form") LampsForm form, BindingResult bindingResult) {
+    return doIfValid(form, bindingResult, (category -> documentRendererService.processImageResponse(internetLabelService.generateInternetLabel(form, form.getEfficiencyRating(), category, ProductMetadata.LAMPS_FULL))));
```
Comment (jamesbarnett91) [inline src/main/java/uk/gov/beis/els/categories/lamps/controller/LampsController.java:78]: Think this needs a `ControllerUtils.validateInternetLabelColour` call first, currently the label colour question isn't being validated on the internet label.

File: src/main/java/uk/gov/beis/els/service/DocumentRendererService.java (hunk 1 above)
Comment (jamesbarnett91) [reply src/main/java/uk/gov/beis/els/service/DocumentRendererService.java:68]: Yeah this seems reasonable. Could you also remove the supplier+model name from `ProcessedEnergyLabelDocument` as well? They don't seem to be used now (a benefit of doing it this way I hadn't thought of), so may as well remove them.

**Review by jamesbarnett91 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/beis/els/categories/lamps/controller/LampsController.java (hunk 2 above)
Comment (matteason) [reply src/main/java/uk/gov/beis/els/categories/lamps/controller/LampsController.java:78]: Oh yes good catch

**Review by jamesbarnett91 — APPROVED** [review]

**Pull request merged**